sentry_sdk/integrations/__init__.py

```diff
@@ -114,21 +116,34 @@ def iter_default_integrations(with_auto_enabling_integrations):
 
 
 def setup_integrations(
-    integrations, with_defaults=True, with_auto_enabling_integrations=False
+    integrations,
+    with_defaults=True,
+    with_auto_enabling_integrations=False,
+    disabled_integrations=None,
 ):
-    # type: (List[Integration], bool, bool) -> Dict[str, Integration]
+    # type: (Sequence[Integration], bool, bool, Optional[Sequence[Integration]]) -> Dict[str, Integration]
     """
     Given a list of integration instances, this installs them all.
 
     When `with_defaults` is set to `True` all default integrations are added
     unless they were already provided before.
+
+    `disabled_integrations` takes precedence over other options. The provided
```

Comment:
I can't really imagine why we'd want to change this specific option interaction but made the change.

Reply:
Yeah, that is fair I guess. But, I don't think we really gain much by guaranteeing how `integrations` and `disabled_integrations` interact, so it would be better not to make a guarantee since it gives us more options